server.js

```diff
@@ -0,0 +1,7 @@
+// This work-around needs for working the VSCode's debugging
+try {
+  module.exports = require('@lifeart/ember-language-server/lib/server').default;
```

Comment:
I'm afraid we always have this path - https://www.runpkg.com/?@lifeart/ember-language-server@2.26.1/lib/index.js, so, there is no way to face "catch"

Reply:
https://npm-explorer.tk/?p=@lifeart/ember-language-server

Reply:
ah, found `.vscodeignore` config, looks like we a good here